.github/workflows/tests-linux.yml

```
@@ -0,0 +1,24 @@
+name: Linux CI
+on:
+  pull_request:
+    paths:
+      - "Formula/libseccomp.rb"
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/ubuntu16.04:master
+    env:
+      HOMEBREW_CORE_GIT_REMOTE: ${{github.event.repository.html_url}}
+      HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+    steps:
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - run: brew test-bot --only-cleanup-before
+
+      - name: Run brew test-bot --only-formulae
+        run: |
+          mkdir ~/bottles
+          cd ~/bottles
+          brew test-bot --only-formulae
```

```suggestion
        run: brew test-bot --only-formulae
```

as we're not using the bottles.